dependency.lic

```diff
@@ -953,15 +962,12 @@ class MoonWatch
     request = Net::HTTP::Get.new(uri.request_uri)
     request['Content-Type'] = 'application/json'
     response = http.request(request)
-    if response.body.nil?
```

Comment:
Wait.. this may have been something rubocop corrected. Let me look before i leap..